### PR TITLE
Update to the latest official release

### PR DESCRIPTION
**Pull request by Hossam-Hawary**
### Upgrade:  **Ruby** version to `3.0.4`

### CODE REVIEW

**Review by ma7dy — CHANGES_REQUESTED** [review]
Why the invoice , department, cities models are in active admin ?

File: .rubocop.yml [hunk 1]
```diff
@@ -434,7 +448,11 @@ Lint/DeprecatedClassMethods:
 Lint/ElseLayout:
   Description: Check for odd code arrangement in an else block.
   Enabled: false
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (Hossam-Hawary) [reply .rubocop.yml]: Resolved

File: .rubocop.yml [hunk 2]
```diff
@@ -205,7 +215,11 @@ Metrics/ClassLength:
   Enabled: false
   CountComments: false
   Max: 100
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (Hossam-Hawary) [reply .rubocop.yml]: Resolved

**Comment by ma7dy** [comment]
👍

**Review by ma7dy — APPROVED** [review]

**Pull request merged**